packages/frontend/src/lib/api.ts

```
@@ -3,20 +3,38 @@
   },
 });
 
+api.interceptors.request.use(
+  config => {
+    const token = localStorage.getItem('token');
+    if (token) {
+      config.headers = config.headers || {};
+      config.headers['Authorization'] = `Bearer ${token}`;
+    } else if (config.headers && 'Authorization' in config.headers) {
+      delete config.headers['Authorization'];
+    }
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)